pyuvsim/utils.py

```diff
@@ -270,24 +267,19 @@ def write_uvdata(
     if noclobber:
         outfile_name = check_file_exists_and_increment(outfile_name)
 
-    if version.parse(pyuvdata.__version__) >= version.parse("2.2.7"):
-        write_kwargs = {'fix_autos': fix_autos}
-    else:
-        write_kwargs = {}
-
     print('Outfile path: ', outfile_name, flush=True)
```

Comment:
I know this isn't a part of this PR, but I don't like bare print statements. Should this be a logging statement? At least then the user can control its output.

Reply:
probably, I'll take a look at it.